docs/management/admin/policy-list.asciidoc

```diff
@@ -2,7 +2,7 @@
 [chapter]
 = Policies
 
-The **Policies** page lists all of the integration policies configured for {endpoint-cloud-sec}. 
+The **Policies** page lists all of the integration policies configured for {elastic-defend}. 
 
 Click on an integration policy's name to configure its settings. For more information on configuring an integration policy, refer to <<configure-endpoint-integration-policy>>.
```

Comment:
Does the following screenshot need to be refreshed?

Reply:
Yes, good catch! Prob needed to be updated back in 8.3, but we'll get it done now. 😉

Reply:
Screenshot updated!